PresentScreenings/Controllers/GoToScreeningDialog.cs

```diff
@@ -97,6 +98,30 @@ public static void UpdateScreeningControls()
                 _labelByfilmScreening[screening].StringValue = screening.ToFilmScreeningLabelString();
             }
         }
+
+        /// <summary>
+        /// Disable resizing of the view containing the given subview.
+        /// </summary>
+        /// <param name="subView"></param>
+        /// <param name="controlName"></param>
+        /// <param name="yFromTop"></param>
+        public void DisableResizing(NSObject subView, string controlName, nfloat yFromTop)
+        {
+            // Get views being constrained.
+            var views = new NSMutableDictionary();
+            views.Add(new NSString(controlName), subView);
+
+            // Define format and assemble constraints (sorry, couln't find an other way).
+            var horzFormat = $"|-[{controlName}]-|";
+            var horzConstraints = NSLayoutConstraint.FromVisualFormat(horzFormat, NSLayoutFormatOptions.None, null, views);
+
+            var vertFormat = $"V:|-{yFromTop}-[{controlName}]";
+            var vertConstraints = NSLayoutConstraint.FromVisualFormat(vertFormat, NSLayoutFormatOptions.None, null, views);
+
+            // Apply constraints.
+            NSLayoutConstraint.ActivateConstraints(horzConstraints);
+            NSLayoutConstraint.ActivateConstraints(vertConstraints);
+        }
```

Comment:
Please remove the `controlName` parameter.
For the use of this method it is of no use at all to have different names per calling dialog.

Reply:
Done.
Plus revised the other parameters, as to generically calculate the distance from window top to subview top.